src/assets/fonts.tsx

```diff
@@ -0,0 +1,11 @@
+import { injectGlobal } from 'styled-components'
+
+injectGlobal`
+    @font-face {
+        font-family: 'artsy-icons';
```

Comment:
@xtina-starr the initial feedback has been addressed so let's merge it. We can deal with the icon-font in a subsequent PR.